static/index.html

```diff
@@ -37,7 +37,7 @@ <h2>Websites in ring</h2>
 					{% for member in members -%}
 						<tr>
 							<td>{{ member.name }}</td>
-							<td><a href="{{ member.website.href }}">{{ member.website.authority }}{%- if member.website.path_and_query != "/" -%}
+							<td><a href="{{ base_addr.scheme }}://{{ base_addr.authority }}{{ base_addr.path_and_query }}visit?member={{ member.website.authority }}">{{ member.website.authority }}{%- if member.website.path_and_query != "/" -%}
```

Comment:
Can we make it skip the tracker if the request has the `DNT: 1` header present? (Yes, it's technically deprecated/non-standard, but browsers still use it.)

Reply:
I'm skeptical that this would be beneficial since we're only incrementing counters and not actually tracking people around.

If we wanted to do this though, the right way would probably be to skip counting them server side if the header is detected rather than changing the homepage links, mostly because changing the links dynamically would be annoying to code and wouldn't apply to everything (like /next or /prev endpoints). Is that something you would be interested in?

Reply:
Yeah, skipping tracking on the request to `/visit` sounds like a better idea.

I can implement that later, so I'll resolve the DNT-related comments for now.

Reply:
> I'm skeptical that this would be beneficial since we're only incrementing counters and not actually tracking people around.

Yeah, I changed my mind and I think it's fine to just always have it on. Eventually, I'll probably try to switch to using a cookie with a UUID instead of the client IP address, as the cookie would be more accurate (tied to the browser, not the network) and also more anonymized (IPs can be used to find location).

Reply:
Ooh, that's a good idea! I will close this comment, but this seems like something you could make as an issue.